### DISCUSSION TITLE
Library initialization failed

### DISCUSSION THREAD

**Discussion by Kolhun** [nlohmann/json#4093] [Q&A]
For the first time I use the library and from the moment of import I ran into a problem that following the instructions gives an error searching for library components. How can this be corrected?

![image](https://github.com/nlohmann/json/assets/36815965/fa065635-a7c5-46b7-9ee2-6bb8ed99df04)

![image](https://github.com/nlohmann/json/assets/36815965/a4d38a91-c7b3-4e82-be06-edb0d9fc4ed5)

**Comment by nlohmann** [comment]
Set the library search path to the include folder and include <nlohmann/json.hpp> instead.

**Reply by Kolhun** [reply]
![image](https://github.com/nlohmann/json/assets/36815965/12edaea6-3b80-4563-b713-28d415fad033)
![image](https://github.com/nlohmann/json/assets/36815965/8c8f8bc8-a0b4-4534-b3f5-0a9dc3303844)

**Reply by Kolhun** [reply]
![image](https://github.com/nlohmann/json/assets/36815965/d36b4743-9373-4a1e-9ae2-284810be85c2)

**Comment by Kolhun** [comment]
![image](https://github.com/nlohmann/json/assets/36815965/7d949c08-0433-466f-ac32-43c05e52cceb)

**Comment by Kolhun** [comment]
![image](https://github.com/nlohmann/json/assets/36815965/51715602-8a57-4786-ae43-ebb733fce634)
The problem is in the file paths, if you change <> and / to "" and \ everything works fine, why is that?And why is there a postscript in the nlohmann path if the files are found together with json.hpp? Do you have a non-working extension? (Corrected the first line)
